ci/actions/windows/run.bat

```diff
@@ -1,20 +1,17 @@
 @echo off
-set exit_code=0
-set count=0
+setlocal
 
-goto %RUN%
-
-:test
-if %count% equ 10 goto rpc_test
 call %BUILD_TYPE%\core_test.exe
 set core_code=%errorlevel%
-set /a count=count+1
-if %core_code% neq 0 goto test
 
-:rpc_test
 call %BUILD_TYPE%\rpc_test.exe
 set rpc_code=%errorlevel%
-echo Core Test %core_code%
-echo RPC Test %rpc_code%
 
-exit /B %core_code%
\ No newline at end of file
+echo Core Test return code: %core_code%
+echo RPC Test return code: %rpc_code%
+
+if %core_code%==0 if %rpc_code%==0 (
```

Comment:
This looks strange but I don't know powershell and I don't know if it is legal. So I am just flagging it to double that it is correct. I was expecting something like:
```
if %core_code%==0 -and %rpc_code%==0 (
```

Reply:
That's actually a good spot, it looked strange to me too, but it seems to be the only way to do `and` construct in windows bat scripts. There is actually a problem with it, it won't handle else condition properly. I pushed a new commit that does it in a different way.

Reply:
Previously the problem was just ignored and only `core_code` result was returned...